ColorPicker/ColorPickerSlider.js

```diff
@@ -178,9 +201,37 @@ const ColorPickerSliderRGB = ({selectedColor, selectedColorHandler, ...props}) =
 	);
 };
 
+ColorPickerSliderRGB.displayName = 'ColorPickerSliderRGB';
+
+ColorPickerSliderRGB.propTypes = {
+	/**
+	 * Indicates the selected color.
+	 *
+	 * @type {String}
+	 * @private
+	 */
+	selectedColor: PropTypes.string,
+
+	/**
+	 * Called when the selected color is modified.
+	 *
+	 * @type {Function}
+	 * @private
+	 */
+	selectedColorHandler: PropTypes.func
+};
+
+/**
+ * A color picker component that allows the user to choose a color by using an HSL color slider.
+ *
+ * @class ColorPickerSliderHSL
+ * @memberof sandstone/ColorPicker
+ * @extends sandstone/Slider.Slider
```

Comment:
on my opinion, a component can extend another component only if the main node is the other component or it is just wrapped in other HTML elements. In this case, I see the usage of Layout/Cell/Row

Reply:
You are right. I will correct the docs.